pages/docs/_meta.json

```diff
@@ -12,7 +12,6 @@
   },
   "essentials": "Effect Essentials",
   "error-management": "Error Management",
-  "interruption-model": "Interruption Model",
```

Comment:
If we move the `interruption-model.mdx` file there are a couple of links in the following files to fix:

- `unexpected-errors.mdx`
- `logging.mdx`

(search for `"interruption-model"` in `./pages/docs`)

Reply:
Fixed the links, thanks for the feedback, sorry I missed that.